sdk/python/tests/compiler/testdata/recur_nested_pipelineloop_cr1.yaml

```diff
@@ -0,0 +1,189 @@
+      timeout: 0s
+    - name: condition-2
+      params:
+      - name: operand1
+        value: $(params.flip-coin-output)
+      - name: operand2
+        value: heads
+      - name: operator
+        value: ==
+      taskSpec:
+        results:
+        - name: outcome
+          description: Conditional task outcome
+        params:
+        - name: operand1
+          type: string
+        - name: operand2
+          type: string
+        - name: operator
+          type: string
+        steps:
+        - script: |-
+            python -c 'import sys
+            input1=str.rstrip(sys.argv[1])
+            input2=str.rstrip(sys.argv[2])
+            try:
+              input1=int(input1)
+              input2=int(input2)
+            except:
+              input1=str(input1)
+            outcome="true" if (input1 $(inputs.params.operator) input2) else "false"
+            f = open("/tekton/results/outcome", "w")
+            f.write(outcome)
+            f.close()' '$(inputs.params.operand1)' '$(inputs.params.operand2)'
+          image: python:alpine3.6
+    - name: condition-5
+      params:
+      - name: operand1
+        value: $(params.flip-coin-output)
+      - name: operand2
+        value: tails
+      - name: operator
+        value: ==
+      taskSpec:
+        results:
+        - name: outcome
+          description: Conditional task outcome
+        params:
+        - name: operand1
+          type: string
+        - name: operand2
+          type: string
+        - name: operator
+          type: string
+        steps:
+        - script: |-
+            python -c 'import sys
+            input1=str.rstrip(sys.argv[1])
+            input2=str.rstrip(sys.argv[2])
+            try:
+              input1=int(input1)
+              input2=int(input2)
+            except:
+              input1=str(input1)
+            outcome="true" if (input1 $(inputs.params.operator) input2) else "false"
+            f = open("/tekton/results/outcome", "w")
+            f.write(outcome)
+            f.close()' '$(inputs.params.operand1)' '$(inputs.params.operand2)'
+          image: python:alpine3.6
+    - name: flip-component
+      taskRef:
+        apiVersion: custom.tekton.dev/v1alpha1
+        kind: PipelineLoop
+        name: nested-recursion-pipeline-76719-graph-flip-component-1
+      params:
+      - name: just_one_iteration
+        value:
+        - '1'
+      - name: flip-coin-output
+        value: $(tasks.flip-coin-3.results.output)
+      - name: maxVal
+        value: $(params.maxVal)
+      when:
+      - input: $(tasks.condition-5.results.outcome)
+        operator: in
+        values:
+        - "true"
+    - name: nested-recursion-pipeline-76719-graph-flip-component-b-3
+      taskRef:
+        apiVersion: custom.tekton.dev/v1alpha1
+        kind: PipelineLoop
+        name: nested-recursion-pipeline-76719-graph-flip-component-b-3
+      params:
+      - name: flip-coin-output
+        value: $(tasks.flip-coin.results.output)
```

Comment:
this value should be `$(params.flip-coin-output)` since flip-coin is not in this pipeline loop.

Reply:
OK, the issue fixed.